rfc/rfc015-node-identity.md

```diff
@@ -0,0 +1,126 @@
+# RFC015 Node identity
+
+|  |  |
+| :--- | :--- |
+| Nuts foundation | W.M. Slakhorst |
+| Request for Comments: 015 | Nedap |
+|  | Nov 2021 |
+
+## Node identity
+
+### Abstract
+
+This RFC describes how to identify a node in the network. Node identification is required for exchanging private data.
+Measures to keep data private can range from encrypting data to sending data over a specific connection.
+The TLS connection and certificate exchange form the basis of the node identification.
+The *Subject Alternative Name* of the certificate is matched against the *serviceEndpoint* of a DID Document service.
```

Comment:
> The *Subject Alternative Name* of the (...)

It might be good to specify which one or all of them as you can have multiple SANs.

Reply:
We might want to add limitations for the SAN as they can contain an e-mail address, DNS name, IP address or URI.

Reply:
I will add the particulars